docs/codestyle.md

```diff
@@ -0,0 +1,58 @@
+# WikiTree code style
+
+## Code style
+
+we aim for consistency, generality, readability and reduction of git diffs. Similar language constructs are formatted with similar rules. Style configuration options are deliberately limited and rarely added. Previous formatting is taken into account as little as possible.
```

Comment:
This paragraph is inspired by [`Black formatter web`](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style)